lms/djangoapps/courseware/tabs.py

```diff
@@ -302,6 +302,9 @@ def get_course_tab_list(request, course):
             if tab.type is not 'courseware':
                 continue
             tab.name = _("Entrance Exam")
+        if tab.type is 'static_tab' and tab.instructors_only and \
```

Comment:
Please use `==` to compare strings, not `is`.  Bonus points for fixing the occurrence on line 302 also :)